backend/src/main/java/ai/verta/modeldb/versioning/CommitDAORdbImpl.java

```diff
@@ -382,18 +390,122 @@ public CommitEntity getCommitEntity(
   }
 
   @Override
```

Comment:
```suggestion
  /**
   * Deleting dataversiosn stored as commits
   * 1. get Repo
   * 2. Iterate through each dataset version
   * 2.1 get Commit
   * 2.2 commit since representing datasetversion verfiy commit belongs to a single repo and match with repo id from 1. 
   * 2.3 get parent commit , since it is dataset assume just single commit
   * 2.4 if commit to be deleted is pointed to by a branch , move branch to parent.
   * 2.5 if commit has children move them to parent
   * 2.5 delete label , tag , attributes for the commit
   */
  @Override
```

Reply:
Added